driver/pelican/demandResponse.go

```diff
@@ -0,0 +1,152 @@
+package main
+
+import (
+	"encoding/xml"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/immesys/spawnpoint/spawnable"
+	bw2 "gopkg.in/immesys/bw2bind.v5"
+)
+
+type ADREventWrapperAPI struct {
+	XMLName xml.Name    `xml:"result"`
+	Success int         `xml:"success"`
+	Message string      `xml:"message"`
+	Event   ADREventAPI `xml:"attribute"`
+}
+
+type ADREventAPI struct {
+	End    string `xml:"OpenADREventEnd"`
+	Start  string `xml:"OpenADREventStart"`
+	Status string `xml:"OpenADRStatus"`
+	Type   string `xml:"OpenADREventType"`
+}
+
+type ADREvent struct {
+	Event_End   int64
```

Comment:
I believe the convention in Go is to use camel case, not underscores, so you'll need to change several of the fields in this struct.